styles/style.scss

```diff
@@ -0,0 +1,154 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$image-path: '../../img';
+$font-path: '../../fonts';
+
+@import '/node_modules/@mozilla-protocol/core/protocol/css/protocol.css';
+@import '/node_modules/@mozilla-protocol/core/protocol/css/includes/lib';
+@import '/node_modules/@mozilla-protocol/core/protocol/css/components/button';
+@import '/node_modules/@mozilla-protocol/core/protocol/css/components/picto';
+@import '/node_modules/@mozilla-protocol/core/protocol/css/components/newsletter-form';
+@import '/node_modules/@mozilla-protocol/core/protocol/css/templates/multi-column';
+
+// Helper classes
+.gradient-bg {
+    background: linear-gradient(rgba(0, 0, 0, 0.8),rgba(0, 0, 0, 0.8)), radial-gradient(#ffa436 0%, #FF298A 50%, #7542e4 100%) right center no-repeat;
+}
+
+.dark-highlight {
+    background-color: $color-black;
+    color: $color-white;
+    padding: 0 $spacing-sm;
+    box-decoration-break: clone;
+    line-height: 1.3;
+}
+
+
+.mzp-c-navigation {
+    background-color: $color-black;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.hero-section {
+    background-image: url("/img/hero-bg.svg"), linear-gradient(90.29deg, rgba(0, 0, 0, 0.5) 72.2%, rgba(0, 0, 0, 0) 99.74%);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-color: $color-black;
+
+    .hero-text-wrapper {
+        max-width: 100%;
+
+        h1 {
+            @include font-size(42px);
+            text-transform: uppercase;
+            font-weight: 500;
+        }
+
+        h2 {
+            @include font-size(64px);
+        }
+    }
+
+    @media #{$mq-md} {
+        .hero-text-wrapper {
+            max-width: 55%;
+        }
+    }
+}
+
+.benefits-section {
+    display: grid;
+    grid-template-columns: [start] 1fr 1fr 1fr [end];
+    column-gap: 24px;
+    row-gap: 32px;
+
+    .benefits-text-wrapper {
+        grid-column: start / end;
+
+        * {
+            max-width: $content-md;
+            margin-left: auto;
+            margin-right: auto;
+        }
+
+        h2 {
+            text-align: center;
+        }
+    }
+
+    .mzp-t-picto-center {
+        grid-column: span 3;
+
+        .mzp-c-picto-heading {
+            @include font-size(24px);
+        }
+    }
+
+    @media #{$mq-md} {
+        .mzp-t-picto-center {
+            grid-column: span 1;
+        }
+    }
+}
+
+
+.question-section .mzp-l-content {
+    background: none;
+}
+
+.teams-text-wrapper {
+    max-width: $content-md;
+    margin: 0 auto $layout-lg;
+
+
+    h2 {
+        text-align: center;
+    }
+}
+
+.teams-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: $spacing-lg;
+}
+
+.collaborate-title {
+    align-self: center;
+}
+
+.faq-section {
+    max-width: 566px;
+
+   h2 {
+        @include font-size(40px);
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 80%;
```

Comment:
**Suggestion**: Instead of playing around with the CSS here, may I suggest to add a `br` in the HTML?

```
<h2><span class="dark-highlight">Frequently Asked
                <br> Questions</span></h2>
```